task/06-conditions-n-loops-tasks.js

```diff
@@ -82,7 +95,14 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,10,10 =>  true
  */
 function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+    if(a+b>c && a+c>b && b+c>a) 
```

Comment:
You can return just boolean expression result.
Need to use indents between operators.

Reply:
Fixed.